composer/core/state.py

```diff
@@ -317,6 +331,14 @@ def algorithms(self):
     def algorithms(self, algorithms: Sequence[Algorithm]):
         self._algorithms[:] = algorithms
 
+    @property
+    def evaluators(self):
+        return self._evaluators
```

Comment:
nit: docstring

Reply:
no docstrings exist for the other getter/setters, i think State in general needs a cleanup that I'll get to in a separate PR.